Src/WitsmlExplorer.Frontend/next.config.js

```diff
@@ -1,19 +1,9 @@
 /* eslint-disable no-undef */
 // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
`// eslint-disable-next-line @typescript-eslint/no-var-requires` can now be removed as well

Reply:
removed